### ISSUE TITLE
Scan all code and docs for any IBM copyright proprietary files 

### ISSUE THREAD

**Issue by dsh33782** [z390development/z390#231]
It has been reported that there are some IBM proprietary files within z390.  They may just be part of MVS files that have been put into public domain by IBM but I think we should do a scan to be sure we are not violating any copyrights.

**Comment by adelosa** [comment]
Is there any way of knowing whether a file with an IBM copyright is public domain vs proprietary? I could take this on, but just not sure what I should be on the lookout for.

**Comment by abekornelis** [comment]
@adelosa the subdirectpry mvs holds MVS 3.8J macros which is public domain. Anything else is IBM's.

**Comment by dsh33782** [comment]
I've searcjhed the repository for "IBM Copyright" and have not found any.  
The MVS 3.8 public domain software donated by IBM in the mvs directory does not have any copyright notice.
I've done some research on the possibility of donating z390 to public domain.  My conclusion for now is to keep it under General Public License.  Here is summary from Red Hat Linux website:
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
What is the difference between the GPL and the public domain?
Updated June 7 2016 at 6:37 PM - English 
It is a common mistake in perception that Free software licenses such as the GPL are the same as any entity in the public domain. However, there are key differences between Free software licenses and public domain software.

Public domain software is one in which all the developers of the software have agreed explicitly to disclaim all their rights granted to them otherwise by copyright laws. This means that any individual or team cannot assert any rights over the software and everyone is allowed to do whatever they want with it.

GPL license on the other hand, is a copyright license which can be only used under the terms of its license. GPL is, however, unique in the fact that terms of the license are designed to protect the freedom of the end users rather than restrict them from doing things.

Examples of software under the GPL license is the Linux kernel, the core of any Linux based systems such as the Fedora Project or Red Hat Enterprise Linux.
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
